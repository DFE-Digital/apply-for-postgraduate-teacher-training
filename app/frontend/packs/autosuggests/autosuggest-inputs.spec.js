import {autosuggestInputs} from "./autosuggest-inputs";

describe("autosuggestInputs", () => {
  it("should return autosuggestInputs", () => {
    expect(autosuggestInputs).toMatchSnapshot();
  });
});
