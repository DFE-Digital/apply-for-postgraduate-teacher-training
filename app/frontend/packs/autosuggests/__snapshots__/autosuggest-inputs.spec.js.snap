// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`autosuggestInputs should return autosuggestInputs 1`] = `
Array [
  Object {
    "containerId": "degree-grade-autosuggest",
    "inputIds": Array [
      "candidate-interface-degree-grade-form-other-grade-field",
      "candidate-interface-degree-grade-form-other-grade-field-error",
    ],
  },
  Object {
    "containerId": "degree-institution-autosuggest",
    "inputIds": Array [
      "candidate-interface-degree-institution-form-institution-name-field",
      "candidate-interface-degree-institution-form-institution-name-field-error",
    ],
  },
  Object {
    "containerId": "degree-subject-autosuggest",
    "inputIds": Array [
      "candidate-interface-degree-subject-form-subject-field",
      "candidate-interface-degree-subject-form-subject-field-error",
    ],
  },
  Object {
    "containerId": "subject-autosuggest-data",
    "inputIds": Array [
      "candidate-interface-other-qualification-details-form-subject-field",
      "candidate-interface-other-qualification-details-form-subject-field-error",
    ],
  },
  Object {
    "containerId": "grade-autosuggest-data",
    "inputIds": Array [
      "candidate-interface-other-qualification-details-form-grade-field",
      "candidate-interface-other-qualification-details-form-grade-field-error",
    ],
    "styles": [Function],
  },
  Object {
    "containerId": "other-uk-qualifications-autosuggest",
    "inputIds": Array [
      "candidate-interface-other-qualification-type-form-other-uk-qualification-type-field",
      "candidate-interface-other-qualification-type-form-other-uk-qualification-type-field-error",
    ],
  },
]
`;
